src/protocol/chromium-bidi.ts

```diff
@@ -121,6 +121,7 @@ export type Command = (
   | ExternalSpecCommand<WebDriverBidiPermissions.PermissionsCommand>
   | ExternalSpecCommand<WebDriverBidiBluetooth.Bluetooth.HandleRequestDevicePrompt>
   | ExternalSpecCommand<WebDriverBidiBluetooth.Bluetooth.SimulateAdapter>
+  | ExternalSpecCommand<WebDriverBidiBluetooth.Bluetooth.DisableSimulation>
```

Comment:
FYI: we can avoid adding those commands manually after this PR is merged: https://github.com/WebBluetoothCG/web-bluetooth/pull/649